pkgs/by-name/fl/flac/package.nix

```diff
@@ -10,17 +10,17 @@
   pkg-config,
   stdenv,
   versionCheckHook,
-  enableManpages ? buildPackages.pandoc.compiler.bootstrapAvailable,
```

Comment:
`git grep enableManpages` does not bring up any results related to this package, so it looks safe to remove again :+1: